ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -1340,6 +1361,12 @@ private void updateClusterMapInHelix(boolean startValidatingClusterManager) thro
     }
     // make sure bootstrap has completed in all dcs (can extend timeout if amount of resources in each datacenter is really large)
     bootstrapLatch.await(15, TimeUnit.MINUTES);
+    for (Map.Entry<String, Exception> entry : exceptions.entrySet()) {
+      if (entry.getValue() != null) {
```

Comment:
If we have multiple entries with exception, we'll throw exception and leave the function. Won't be able to print all the warnings.
Is this expected?

Reply:
that's fine, multiple exceptions can be resolved one at a time.